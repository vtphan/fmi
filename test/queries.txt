abracadabra
abracadabr
abracadab
abracada
abracad
abraca
abrac
abra
abr
ab
a
bracadabra
bracadabr
bracadab
bracada
bracad
braca
brac
bra
br
b
racadabra
racadabr
racadab
racada
racad
raca
rac
ra
r
acadabra
acadabr
acadab
acada
acad
aca
ac
cadabra
cadabr
cadab
cada
cad
ca
c
adabra
adabr
adab
ada
ad
dabra
dabr
dab
da
d
$$$
$$
$
qabra
qabr
qab
qa
q
xyz
xy
z
