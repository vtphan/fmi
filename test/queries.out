0 
0 
0 
0 
0 
0 
0 
7 0 
7 0 
7 0 
10 7 0 3 5 
1 
1 
1 
1 
1 
1 
1 
8 1 
8 1 
8 1 
2 
2 
2 
2 
2 
2 
2 
9 2 
9 2 
3 
3 
3 
3 
3 
3 
3 
4 
4 
4 
4 
4 
4 
4 
5 
5 
5 
5 
5 
6 
6 
6 
6 
6 
na
na
na
na
na
na
na
na
na
na
na
